inspirehep/modules/workflows/workflows/article.py

```diff
@@ -359,11 +358,7 @@
 
 
 STORE_RECORD_AND_ROOT = [
-    IF_ELSE(
-        is_marked('is-update'),
-        update_record,
-        store_record
-    ),
+    store_record,
     store_root,
```

Comment:
If `store_root` makes no sense without `store_record`, maybe it's better to join both tasks and do everything together.